internal/trace/trace.go

```diff
@@ -32,16 +32,15 @@ import (
 )
 
 const (
-	telemetryPlatformTracingOpenCensus    = "opencensus"
-	telemetryPlatformTracingOpenTelemetry = "opentelemetry"
-	telemetryPlatformTracingVar           = "GOOGLE_API_GO_EXPERIMENTAL_TELEMETRY_PLATFORM_TRACING"
+	TelemetryPlatformTracingOpenCensus    = "opencensus"
+	TelemetryPlatformTracingOpenTelemetry = "opentelemetry"
+	TelemetryPlatformTracingVar           = "GOOGLE_API_GO_EXPERIMENTAL_TELEMETRY_PLATFORM_TRACING"
 )
 
 var (
-	// TODO(chrisdsmith): Should the name of the OpenTelemetry tracer be public and mutable?
-	openTelemetryTracerName     string = "cloud.google.com/go"
-	openTelemetryTracingEnabled bool   = strings.EqualFold(strings.TrimSpace(
-		os.Getenv(telemetryPlatformTracingVar)), telemetryPlatformTracingOpenTelemetry)
+	OpenTelemetryTracerName     string = "cloud.google.com/go"
```

Comment:
Random thought. Will exporting this not cause problems?
For ex, if spanner handwritten client overrides this "cloud.google.com/go/spanner", then the same change will get affected to other for ex: bigquery etc.

Am i missing something here?

Reply:
Yes, these are not meant to be changed as it would be a global change across multiple clients. Maybe we need to understand your usercase more

Reply:
Good call, let's not export this, thanks!

Reply:
Actually, I propose we keep it exported but make it a `const`. It was a `var` only because originally I thought it might need to be mutable. That was incorrect.